cla-backend/cla/models/docusign_models.py

```diff
@@ -408,7 +409,17 @@ def check_and_prepare_employee_signature(project_id, company_id, user_id) -> dic
 
         # Assume this company is the user's employer.
         # TODO: DAD - we should check to see if they already have a company id assigned
-        user.set_user_company_id(str(company_id))
+        if user.get_user_company_id() != company_id:
+            user.set_user_company_id(str(company_id))
+            Event.create_event(
+                event_type=EventType.UserAssociatedWithCompany,
+                event_company_id=company_id,
+                event_project_id=project_id,
+                event_user_id=user.get_user_id(),
+                event_data='user {} associated himself with company {}'.format(user.get_user_name(),company.get_company_name()),
+                contains_pii=True,
```

Comment:
I see theres a contain_pii attribute that wasnt defined in the create_event function and hasnt been updated as such